packages/core/docs/modules/fixtures.md

```diff
@@ -0,0 +1,17 @@
+[@hoprnet/hopr-core](../README.md) / [Exports](../modules.md) / fixtures
+
+# Module: fixtures
```

Comment:
We can safely ignore all fixtures and `.spec` files

Reply:
Both are now out of the generated docs.